src/session.ts

```diff
@@ -348,21 +376,21 @@ class Session extends ServiceObject {
           method: 'executeSql',
           reqOpts,
         },
-        callback);
+        callback!);
   }
   /**
    * Create a Transaction object.
    *
    * @throws {Error} If an ID is not provided.
    *
-   * @param {string} id The id of the transaction.
+   * @param {object} options The options of the transaction.
    * @return {Transaction} A Transaction object.
    *
    * @example
    * const transaction = database.transaction('transaction-id');
    */
-  transaction(id) {
```

Comment:
That's weird.  Was this a bug in the jsdoc?

Reply:
Yes, we can see inside this function, it returning Transaction constructor and it will take first param as session and second as TransactionOptions.